cypress/integration/favoritesPageView.js

```diff
@@ -0,0 +1,53 @@
+describe('Favorites page view', () => {
```

Comment:
Very thorough testing! Also just an FYI you only need the cy. in front of the first command under the "it" line for Cypress. Each cy. after that is redundant but won't hurt.